datalad_osf/git_remote.py

```diff
@@ -55,6 +55,8 @@ def __init__(self,
           Stream for logging.
         """
         self.parsed_url = urlparse(url)
+        if self.parsed_url.path:  # project urls have no path
+            raise RuntimeError("Invalid OSF project URL: %s" % url)
```

Comment:
This message is not very useful. It should convey what users need to do.

Reply:
Done. Also check PR #143.